Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ruby:3.2.2
+
+RUN curl -sL https://deb.nodesource.com/setup_20.x | bash - && \
```

Comment:
It seems `curl -sL https://deb.nodesource.com/setup_20.x | bash -` is deprecated, according to https://github.com/nodesource/distributions/discussions/1639. The new, approved method involves a bunch more commands. You can find it at that URL in multiple answers, but it's something like this:
```
# Install NodeJs
ARG NODE_MAJOR=20
RUN apt-get update && \
   apt-get install -y ca-certificates curl gnupg && \
   curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/trusted.gpg.d/nodesource.gpg && \
   echo "deb [signed-by=/etc/apt/trusted.gpg.d/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | tee /etc/apt/sources.list.d/nodesource.list && \
   apt-get update && \
   apt-get install -y nodejs
# End Install
```
I have tried it. What do you think?

Reply:
i didn't get the outdated warning in CI, but i changed it to the recommended installation way